docs/source/getting-started/example-project.rst

```diff
@@ -0,0 +1,66 @@
+
+To generate a random dataset, a label map, and attributions for the samples in the dataset (the attributions are generated randomly as well, and are not from a real classifier), the ``make_test_data.py`` script can be used as follows:
+
+.. code-block:: console
+
+    mkdir -p test-project
+    .venv/bin/python make_test_data.py \
+        test-project/input.h5 \
+        test-project/attribution.h5 \
+        test-project/label-map.json
+
+A meta-analysis of the data can then be generated using the ``meta_analysis.py`` script, which takes the attributions as input and runs them through a CoRelAy meta-analysis pipeline. The pipeline generates spectral embeddings, t-SNE embeddings and UMAP embeddings, and clusters them using agglomerative clustering, DBSCAN clustering, HDBSCAN clustering, and k-nearest neighbor clustering with various different parameters.
+
+.. code-block:: console
+
+    .venv/bin/python meta_analysis.py \
+        test-project/attribution.h5 \
+        test-project/analysis.h5 \
+        --label-map test-project/label-map.json
+
+Finally, to generate a project file for the randomly generated data, the ``make_project.py`` script can be used like so:
+
+.. code-block:: console
+
+    .venv/bin/python make_project.py \
+        test-project/input.h5 \
+        test-project/attribution.h5 \
+        test-project/analysis.h5 \
+        test-project/label-map.json \
+        --project-name 'Test Project' \
+        --dataset-name 'Random Data' \
+        --model-name 'No Model' \
+        --attribution-name 'Random Attribution' \
+        --analysis-name 'Spectral Analysis' \
+        --output test-project/project.yaml
+
+The resulting project can then be opened in ViRelAy using the following command:
+
+.. code-block:: console
+
+    python -m virelay test-project/project.yaml
```

Comment:
```suggestion
    $ .venv/bin/python -m virelay test-project/project.yaml
```

Reply:
Why not activate the virtual environment?

Reply:
Technically activating is okay, although the rest of the documentation uses the python binary directly. Also I think using the python binary directly is preferable, because it is context-free (continue the example at any point without the need to reactivate the environment), which can prevent obscurities for some users.

Reply:
Okay, that sounds reasonable. I'll change it.